src/multitap.c

```diff
@@ -7,7 +7,7 @@
 #include <stdlib.h>
 #include <stdio.h>
 
-#define MT_BUFFER_MS 25
+#define MT_BUFFER_MS 15
```

Comment:
I haven't played with these changes so I'm not sure if they make things better or worse, and this also tends to be a subjective experience, so the current settings work well for some but not others, etc... I would want to have this vetted through a decent sized group. Maybe it can go in a PR separate from this one if you meant to include it and think it's better than the current settings?